Content.Client/Shuttles/UI/RadarControl.cs

```diff
@@ -221,7 +222,9 @@ protected override void Draw(DrawingHandleScreen handle)
             var gridMatrix = gridXform.WorldMatrix;
             Matrix3.Multiply(in gridMatrix, in offsetMatrix, out var matty);
 
-            if (ShowIFF)
+            if (ShowIFF &&
+                (!_entManager.TryGetComponent<IFFComponent>(grid.GridEntityId, out var iff) ||
+                (iff.Flags & IFFFlags.HideLabel) == 0x0))
```

Comment:
mfw i quickly did smth like this for OR and you did it better and now i get merg conflicts.

One request, can you allow the IFFComponent to set the draw color.

Reply:
Thought I did this already but turns out I'd only done it for docks.